create_bumblebee_app.py

```diff
@@ -1,6 +1,6 @@
 from helpers import bumblebee_root, python3_path
 
-with open("bumblebee_app", "a+") as file:
+with open("bumblebee_app", "w") as file:
```

Comment:
Was it appending to the file before this change?

Reply:
Yhup I believe so